less/lists.less

```diff
@@ -111,7 +111,7 @@
     margin: auto;
     height: 100%;
     width: 100%;
-    font-size: 0.84em;
+    font-size: 0.90em;
```

Comment:
Font size need a reproper rework ... I have also plan to rework mediaqueries properly and avoid some test, I can try to do on font-size